<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="Tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRgb"/>
    <addaction name="separator"/>
    <addaction name="actionBW_component"/>
    <addaction name="separator"/>
    <addaction name="actionSquare"/>
    <addaction name="separator"/>
    <addaction name="actionPentagramm_2"/>
    <addaction name="separator"/>
    <addaction name="actionCircle_2"/>
    <addaction name="separator"/>
    <addaction name="actionRotate"/>
    <addaction name="separator"/>
    <addaction name="actionSwap_4_pieces_in_2"/>
    <addaction name="separator"/>
    <addaction name="actionCollage"/>
    <addaction name="separator"/>
    <addaction name="actionStep_back_2"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionCreate"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionInfo_about_File"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_program_2"/>
   </widget>
   <widget class="QMenu" name="menuChoosing_color">
    <property name="title">
     <string>Tools format</string>
    </property>
    <addaction name="actionChoosing_color"/>
    <addaction name="separator"/>
    <addaction name="actionLine_thickness"/>
    <addaction name="separator"/>
    <addaction name="actionFill_color"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="Tools"/>
   <addaction name="menuChoosing_color"/>
   <addaction name="menuInfo"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout_program">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="actionRgb">
   <property name="text">
    <string>Rgb</string>
   </property>
  </action>
  <action name="actionReverse">
   <property name="text">
    <string>Reverse</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="text">
    <string>Square</string>
   </property>
  </action>
  <action name="RotateFullPicture">
   <property name="text">
    <string>Full picture</string>
   </property>
  </action>
  <action name="RotatePartOfPicture">
   <property name="text">
    <string>Part of picture</string>
   </property>
  </action>
  <action name="actionInfo_about_File">
   <property name="text">
    <string>About file</string>
   </property>
  </action>
  <action name="actionAbout_program_2">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="actionChoosing_color">
   <property name="text">
    <string>Line color</string>
   </property>
  </action>
  <action name="actionLine_thickness">
   <property name="text">
    <string>Line thickness</string>
   </property>
  </action>
  <action name="actionFill_color">
   <property name="text">
    <string>Fill color</string>
   </property>
  </action>
  <action name="actionRadius_and_center_coordinates">
   <property name="text">
    <string>Radius and center coordinates</string>
   </property>
  </action>
  <action name="actionSquare_coordinates">
   <property name="text">
    <string>Square coordinates</string>
   </property>
  </action>
  <action name="actionStep_back">
   <property name="text">
    <string>Step back</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCircle_2">
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionStep_back_2">
   <property name="text">
    <string>Step back</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionBW_component">
   <property name="text">
    <string>BW component</string>
   </property>
  </action>
  <action name="actionSmallest_rectangle">
   <property name="text">
    <string>Smallest rectangle</string>
   </property>
  </action>
  <action name="actionCollage">
   <property name="text">
    <string>Collage</string>
   </property>
  </action>
  <action name="actionSwap">
   <property name="text">
    <string>Swap 4 pieces in rectangle</string>
   </property>
  </action>
  <action name="actionSwap_4_pieces_in_circle">
   <property name="text">
    <string>Swap 4 pieces in circle</string>
   </property>
  </action>
  <action name="actionSwap_4_pieces_in">
   <property name="text">
    <string>Swap 4 pieces in...</string>
   </property>
  </action>
  <action name="actionSwap_4_pieces_in_2">
   <property name="text">
    <string>Swap 4 pieces in...</string>
   </property>
  </action>
  <action name="actionPentagramm_2">
   <property name="text">
    <string>Pentagramm</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
